applications/salk-portal/src/components/ExplorerSidebarDialog.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import {
+  Box,
+  makeStyles,
+  Grid,
+  Typography,
+  Link,
+  Container
+} from "@material-ui/core";
+import { switchActiveColor, headerBorderColor } from "../theme";
+import LOGO from "../assets/images/logo_icon.svg"
+import Modal from "./common/BaseDialog";
+
+const useStyles = makeStyles(() => ({
+  learnMore: {
+    '& .details': {
+      flexGrow: 1,
+
+      '& .detail-block + .detail-block': {
+        marginTop: '1.5rem',
+      },
+
+      '& .logo-container': {
+
+        '& .logo': {
```

Comment:
@paularah if you use Avatar you would not need to add these extra styles for structuring